CMakeLists.txt

```diff
@@ -95,3 +95,15 @@ TARGET_LINK_LIBRARIES(picoquic_ct picoquic-core
 )
 
 SET(TEST_EXES picoquic_ct)
+
+# get all project files for formatting
+file(GLOB_RECURSE CLANG_FORMAT_SOURCE_FILES *.c *.h)
+
+# Adds clangformat as target that formats all source files
+add_custom_target(
+    clangformat
+    COMMAND clang-format
+    -style=Webkit
+    -i
+    ${CLANG_FORMAT_SOURCE_FILES}
+)
```

Comment:
That's cool. So "make clangformat" would redo the indentations, correct?

Reply:
If you have `clang-format` installed, it will always format the source code in the Webkit style. So, you can write as you want and before committing, you just need to execute that once and everything is formatted. :)